.github/namespace-scope-int-workflow.yml

```diff
@@ -0,0 +1,139 @@
+name: Namespace-scope Operator Integration Test WorkFlow
+on:
+  schedule:
+    - cron: "0 02 * * WED,SUN"
+jobs:
+  int-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        test: [appframeworks1, appframeworkc3, appframeworkm4, secret, smartstore, monitoringconsole, scaling, crcrud, licensemaster]
```

Comment:
Please change this to name on new line to be consistent with feature-appframework.